### PR TITLE
PIL-433 PIL 459 View Contact details from Read subscription API

### PR DESCRIPTION
**Pull request by javed-iqbal1**

### CODE REVIEW

File: app/views/subscriptionview/manageAccount/AddSecondaryContactView.scala.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form,  errorLinkOverrides = Map("value" -> "value_0")))
+    }
+        <span class="govuk-caption-l">@messages("taskList.task.contact.heading", contactName)</span>
+        <h1 class="govuk-heading-l">@messages("addSecondaryContact.heading", contactName)</h1>
+
+        @govukRadios(
+            RadiosViewModel.yesNo(
+                field = form("value"),
+                legend = LegendViewModel("")
```
Comment (AKKUM) [inline app/views/subscriptionview/manageAccount/AddSecondaryContactView.scala.html:43]: Why Legend View Model is blank?

File: app/views/subscriptionview/manageAccount/ContactEmailAddressView.scala.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode, UserName:String ="")(implicit request: Request[_], appConfig: FrontendAppConfig, messages: Messages)
```
Comment (AKKUM) [inline app/views/subscriptionview/manageAccount/ContactEmailAddressView.scala.html:30]: Please remove ==""

File: app/views/subscriptionview/manageAccount/SecondaryContactEmailView.scala.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode, secondaryContactName: String = "")(implicit request: Request[_], appConfig: FrontendAppConfig, messages: Messages)
```
Comment (AKKUM) [inline app/views/subscriptionview/manageAccount/SecondaryContactEmailView.scala.html:30]: same here

File: app/views/subscriptionview/manageAccount/SecondaryTelephonePreferenceView.scala.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton
+    )
+
+@(form: Form[_], mode: Mode, userName: String= "")(implicit request: Request[_], appConfig: FrontendAppConfig, messages: Messages)
```
Comment (AKKUM) [inline app/views/subscriptionview/manageAccount/SecondaryTelephonePreferenceView.scala.html:28]: Here

Comment (javed-iqbal1) [reply app/views/subscriptionview/manageAccount/SecondaryTelephonePreferenceView.scala.html]: updated

**Review by AKKUM — APPROVED** [review]

**Pull request merged**